### PR TITLE
Enable Trash, delete images and check if they are moved to trash automatically

### PR DESCRIPTION
**Pull request by preethisn**
# Description - Enable "rbd_move_to_trash_on_remove" config setting, delete images and check if they are moved to trash automatically

- Added a new module rbd_trash.py and test case is called in tier-2_rbd_regression.yaml

### CODE REVIEW

**Comment by preethisn** [comment]
logs attached - http://magna002.ceph.redhat.com/ceph-qe-logs/Preethi/cephci-run-QXN7HM/Check_trash_if_the_deleted_images_are_moved_to_trash_automatically_0.log

File: tests/rbd/rbd_trash.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+
+    Test case covered -
+    CEPH-83573297 - Enable "rbd_move_to_trash_on_remove" config setting, delete images
+    and check if they are moved to trash automatically.
+
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+       (At least with 64 pgs)
```
Comment (VasishtaShastry) [inline tests/rbd/rbd_trash.py:26]: Can we please remove this line as it is not mandatory have 64 PGs for our TC ?

File: tests/rbd/rbd_trash.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    Test Case Flow:
+    1. Create a pool and an Image
+    2. generate IO in images
+    3. Enable "rbd_move_to_trash_on_remove" in client config
+    4. Delete images and verify the deleted images are moved to trash automatically
+
+    """
+    log.info("Running Trash function")
+    log.info("executing *****")
```
Comment (VasishtaShastry) [inline tests/rbd/rbd_trash.py:38]: Can we please merge these lines and have single info at the start of the module ?

**Comment by VasishtaShastry** [comment]
(Removed request to review on Gopi)

File: suites/pacific/rbd/tier-2_rbd_regression.yaml [hunk 3]
```diff
@@ -86,3 +87,10 @@ tests:
       module: rbd_clonev2.py
       name: clones creation with v2clone format
       polarion-id: CEPH-83573308, CEPH-83573307, CEPH-83573309
```
Comment (Manohar-Murthy) [inline suites/pacific/rbd/tier-2_rbd_regression.yaml:89]: Please combine the tests and have one polarion id, we can discuss further if needed.

Comment (preethisn) [reply suites/pacific/rbd/tier-2_rbd_regression.yaml]: done

Comment (preethisn) [reply suites/pacific/rbd/tier-2_rbd_regression.yaml]: CEPH-83573309 - This test covers all functionality of V2clone and has steps of CEPH-883573308 CEPH-83573307

**Review by Manohar-Murthy — APPROVED** [review]
looks good.

**Pull request merged**